src/Pos/Block/Network/Logic.hs

```diff
@@ -289,7 +282,7 @@ requestHeaders mgh peerId origTip _ conv = do
 -- First case of 'handleBlockheaders'
 handleRequestedHeaders
     :: forall ssc m.
-       (WorkMode ssc m)
+       (SscWorkersClass ssc, WorkMode ssc m)
```

Comment:
I think GHC asked for it. Not sure. I'll check and if it's not needed, I'll remove it.